content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+@griffiths_language_2007 were the first to show that the analytical structure of iterated learning with uniform Bayesian agents can correspond, depending on the way the agents produce new iterations, to well-known statistical inference methods (Gibbs sampling and a flavour of the EM algorithm).
+In such a setup, iterated learning predictably converges towards distributions determined by the internal prior distributions of agents (i.e. their inference bias).
+As a consequence, in those analytically derived situations one can straightforwardly predict the final distributions that should evolve under iterated learning, a fact that @kalish_iterated_2007 verified with humans in a function learning task.
+@griffiths_using_2008 further exploit this result by using it in the reverse direction:
+since the outcome of iterated learning, for specific setups, is predictable on the basis of participants' priors, one can use such experiments to investigate the inductive biases of participants.
+The authors confirmed this, showing that the method infers well-known participant biases in category learning tasks.
+@griffiths_theoretical_2008 then explored the relevance of these findings for the study of cultural evolution, showing in particular that individual cognitive biases can have significant effects on long-term cultural evolution.
+@reali_evolution_2009 further related those results to the evolution of vocabulary, showing that they are consistent with experimental cases of word-meaning mapping regularisation.
+Finally @perfors_language_2014, through analytical derivation and experimental confirmation, reintroduced the impact of the external world in those results;
+the authors showed that the structure of referents (i.e., the external world) will also play a role in the final evolved language, provided it has an effect on the choice of items people actually talk about (versus, the choice of items talked about only depends on the language itself).
+
+@scott-phillips_language_2010 and @tamariz_cultural_2016 provide further reviews of the iterated learning literature, and @galantucci_experimental_2012 and @roberts_investigating_2017 offer reviews of experimental semiotics.
+An interesting and important development in recent works is the reintroduction of pragmatics into theoretical questions.
+@scott-phillips_pragmatics_2017, in particular, reaffirms the central role of pragmatics in the creation and understanding of meaning in context, and argues for a much stronger focus on the evolution of pragmatics itself, that is, as he envisions it, on the evolution of *ostensive communication*.
+Let me close this review by noting that there is an increasing convergence both in the literature and in empirical questions, of the cultural evolution and language evolution fields;
+the intersection of questions from the two fields is likely to push theoretical issues forward.
+
+\add{sperber-beyond-2015}
+
+
+#### Digital media
```

Comment:
Majeurs, ce serait l'objet d'une review, mais tu peux déjà faire ton marché ici:

Thomas W. Valente, 1996, "Social Network Thresholds in the Diffusion of Innovations"

Jacob Goldenberg, Barak Libai, Eitan Muller, 2001, "Talk of the Network: A Complex Systems Look at the Underlying Process of Word-of-Mouth"

David Kempe, Jon Kleinberg, Eva Tardos, 2003, "Maximizing the Spread of Influence through a Social Network"

David Kempe, Jon Kleinberg, Eva Tardos, 2003, "Maximizing the Spread of Influence through a Social Network"

Eric Abrahamson, Lori Rosenkopf, 1997, "Social Network Effects on the Extent of Innovation Diffusion: A Computer Simulation"

Frédéric Deroïan, 2002, "Formation of social networks and diffusion of innovations"

C. de Kerchove, G. Krings, R. Lambiotte, V. D. Blondel, P. Van Dooren, 2009, "The Role of Second Trials in Cascades of Information over Networks"

J. P. Onnela, Felix Reed-Tsochas, 2010, "The Spontaneous Emergence of Social Influence in Online Systems"

Lilian Weng, Alessandro Flammini, Alessandro Vespignani, Filippo Menczer, "Competition among memes in a world with limited attention", 2012

Daniel M. Romero, Brendan Meeder, Jon Kleinberg, 2011, "Differences in the Mechanics of Information Diffusion Across Topics: Idioms, Political Hashtags, and Complex Contagion on Twitter"

Meeyoung Cha, Alan Mislove, Krishna P. Gummadi, 2009, "A Measurement-driven Analysis of Information Propagation in the Flickr Social Network"


et tiens en passant, des choses non atomiques mais très scalaires:

Konstantin Klemm, Victor M. Eguiluz, Raul Toral, Maxi San Miguel, 2005, "Globalization, polarization and cultural drift"

Frédéric Amblard, Guillaume Deffuant, 2004, "The role of network topology on extremism propagation with the relative agreement opinion dynamics"

voire les anciens 

George Kingsley Zipf, 1946, "Some determinants of the Circulation of Information"